[2024-05-19T20:26:49.783+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-19T20:26:49.805+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: silver_s3.process_parquet_files scheduled__2023-05-13T00:00:00+00:00 [queued]>
[2024-05-19T20:26:49.826+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: silver_s3.process_parquet_files scheduled__2023-05-13T00:00:00+00:00 [queued]>
[2024-05-19T20:26:49.830+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-05-19T20:26:49.866+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): process_parquet_files> on 2023-05-13 00:00:00+00:00
[2024-05-19T20:26:49.876+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=743) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-19T20:26:49.890+0000] {standard_task_runner.py:63} INFO - Started process 768 to run task
[2024-05-19T20:26:49.882+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'silver_s3', 'process_parquet_files', 'scheduled__2023-05-13T00:00:00+00:00', '--job-id', '262', '--raw', '--subdir', 'DAGS_FOLDER/silver_s3.py', '--cfg-path', '/tmp/tmpbjcppgmr']
[2024-05-19T20:26:49.894+0000] {standard_task_runner.py:91} INFO - Job 262: Subtask process_parquet_files
[2024-05-19T20:26:50.033+0000] {task_command.py:426} INFO - Running <TaskInstance: silver_s3.process_parquet_files scheduled__2023-05-13T00:00:00+00:00 [running]> on host 620090466bc8
[2024-05-19T20:26:50.215+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='silver_s3' AIRFLOW_CTX_TASK_ID='process_parquet_files' AIRFLOW_CTX_EXECUTION_DATE='2023-05-13T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-05-13T00:00:00+00:00'
[2024-05-19T20:26:50.224+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-19T20:26:50.324+0000] {base.py:84} INFO - Using connection ID 'aws_s3' for task execution.
[2024-05-19T20:26:50.326+0000] {connection_wrapper.py:384} INFO - AWS Connection (conn_id='aws_s3', conn_type='aws') credentials retrieved from login and password.
[2024-05-19T20:26:55.492+0000] {logging_mixin.py:188} INFO - No .parquet files found.
[2024-05-19T20:26:55.495+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-19T20:26:55.497+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-19T20:26:55.513+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=silver_s3, task_id=process_parquet_files, run_id=scheduled__2023-05-13T00:00:00+00:00, execution_date=20230513T000000, start_date=20240519T202649, end_date=20240519T202655
[2024-05-19T20:26:55.563+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-19T20:26:55.572+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-19T20:26:55.573+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-05-19T20:36:37.536+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-19T20:36:37.552+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: silver_s3.process_parquet_files scheduled__2023-05-13T00:00:00+00:00 [queued]>
[2024-05-19T20:36:37.565+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: silver_s3.process_parquet_files scheduled__2023-05-13T00:00:00+00:00 [queued]>
[2024-05-19T20:36:37.566+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-05-19T20:36:37.585+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): process_parquet_files> on 2023-05-13 00:00:00+00:00
[2024-05-19T20:36:37.589+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1874) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-19T20:36:37.592+0000] {standard_task_runner.py:63} INFO - Started process 1884 to run task
[2024-05-19T20:36:37.593+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'silver_s3', 'process_parquet_files', 'scheduled__2023-05-13T00:00:00+00:00', '--job-id', '375', '--raw', '--subdir', 'DAGS_FOLDER/silver_s3.py', '--cfg-path', '/tmp/tmpifidyib_']
[2024-05-19T20:36:37.596+0000] {standard_task_runner.py:91} INFO - Job 375: Subtask process_parquet_files
[2024-05-19T20:36:37.626+0000] {task_command.py:426} INFO - Running <TaskInstance: silver_s3.process_parquet_files scheduled__2023-05-13T00:00:00+00:00 [running]> on host 620090466bc8
[2024-05-19T20:36:37.678+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='silver_s3' AIRFLOW_CTX_TASK_ID='process_parquet_files' AIRFLOW_CTX_EXECUTION_DATE='2023-05-13T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-05-13T00:00:00+00:00'
[2024-05-19T20:36:37.680+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-19T20:36:37.701+0000] {base.py:84} INFO - Using connection ID 'aws_s3' for task execution.
[2024-05-19T20:36:37.702+0000] {connection_wrapper.py:384} INFO - AWS Connection (conn_id='aws_s3', conn_type='aws') credentials retrieved from login and password.
[2024-05-19T20:36:47.003+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-19T20:36:47.012+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/silver_s3.py", line 69, in process_parquet_files
    os.remove(file_path)
FileNotFoundError: [Errno 2] No such file or directory: '/tmp/disney_data_20240518T000000.parquet'
[2024-05-19T20:36:47.044+0000] {taskinstance.py:1206} INFO - Marking task as UP_FOR_RETRY. dag_id=silver_s3, task_id=process_parquet_files, run_id=scheduled__2023-05-13T00:00:00+00:00, execution_date=20230513T000000, start_date=20240519T203637, end_date=20240519T203647
[2024-05-19T20:36:47.065+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 375 for task process_parquet_files ([Errno 2] No such file or directory: '/tmp/disney_data_20240518T000000.parquet'; 1884)
[2024-05-19T20:36:47.098+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-05-19T20:36:47.112+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-19T20:36:47.113+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
